src/string_input_popup.cpp

```diff
@@ -490,6 +505,12 @@ const std::string &string_input_popup::query_string( const bool loop, const bool
             if( _position < static_cast<int>( ret.size() ) ) {
                 ret.erase( _position, 1 );
             }
+        } else if( action == "HELP_KEYBINDINGS" ) {
+            ctxt->display_menu();
+        } else if( action == "SCROLL_INFOBOX_UP" ) {
+            description_view->scroll_up();
+        } else if( action == "SCROLL_INFOBOX_DOWN" ) {
+            description_view->scroll_down();
```

Comment:
> If there's anywhere where you need to type a < or > into a string_input_popup, this will prevent you from doing so

Use `PPAGE` and `NPAGE` by default then? You can override the default by adding `"SCROLL_INFOBOX_UP"` and `"SCROLL_INFOBOX_DOWN"` to `data/keybindings.json` with `"category": "STRING_INPUT"`.

Reply:
Add mouse scrolling support perhaps? For people who do not use a mouse and do not have page up and down keys, they can rebind the keys as needed, but I think it's better to avoid using character keys in a text input field as a hotkey.